src/services/auth/authService.ts

```
@@ -79,6 +123,83 @@ function generateOtpCode(): string {
+function validateAdminScopes(scopes: string[]): string[] {
+  if (!Array.isArray(scopes) || scopes.length === 0) {
+    return [];
+  }
+  const allowed = new Set<string>(ADMIN_SCOPES);
+  return scopes.filter((scope) => allowed.has(scope));
+}
+
+async function publishOtp(channel: "sms" | "email", to: string, code: string) {
+  const ch = getRabbitMQChannel();
+  await ch.assertQueue(QUEUES.OTP_SEND, { durable: true });
+  ch.sendToQueue(QUEUES.OTP_SEND, Buffer.from(JSON.stringify({ channel, to, code })), {
```

ESLint

[error] Replace `QUEUES.OTP_SEND,·Buffer.from(JSON.stringify({·channel,·to,·code·})),` with `⏎····QUEUES.OTP_SEND,⏎····Buffer.from(JSON.stringify({·channel,·to,·code·})),⏎···`

(prettier/prettier)